tla/consensus/ccfraft.tla

```diff
@@ -981,20 +981,23 @@ AppendEntriesAlreadyDone(i, j, index, m) ==
                 IF membershipState = RetirementSigned /\ commitIndex' > RetirementIndex(i) 
                 THEN RetirementCompleted 
                 ELSE @]
-    /\ isNewFollower' = [isNewFollower EXCEPT ![i] = FALSE]
```

Comment:
Keeping the spec up-to-date with the code. We now only reset `isNewFollower` when processing a conflict, it is unchanged on other AEs.